src/gui/syncrunfilelog.cpp

```diff
@@ -32,7 +32,10 @@ void SyncRunFileLog::start(const QString &folderPath)
 {
     const qint64 logfileMaxSize = 10 * 1024 * 1024; // 10MiB
 
-    const QString logpath = QStandardPaths::writableLocation(QStandardPaths::AppDataLocation);
+    const QString logpath =
+        !Utility::isWindows()
+        ? QStandardPaths::writableLocation(QStandardPaths::AppConfigLocation)
+        : QStandardPaths::writableLocation(QStandardPaths::AppDataLocation);
```

Comment:
why do we need to special case windows ?

Reply:
Because of the way how the configPath is generated in [src/libsync/configfile.cpp#L366](https://github.com/kaikli/nextcloud-desktop-client/blob/fix-7654/src/libsync/configfile.cpp#L366) and Windows stores everything to the data location instead of the config location.

But I think we can also use the way I have solved it in my second PR [!7665](https://github.com/nextcloud/desktop/pull/7665/files#diff-6d359e2f56dc9b47e9108fd1467f60f3a05ba084b88d028218b054d71b7dc03d).

Still I would prefer the way I suggested with !7665